data/json/mapgen/house/2storymodern02.json

```diff
@@ -0,0 +1,197 @@
+    "weight": 250,
+    "method": "json",
+    "object": {
+      "palettes": [ "standard_domestic_palette", "standard_domestic_basement_palette" ],
+      "fill_ter": "t_thconc_floor",
+      "rows": [
+        "^^^^^^^^^^^^^^^^^^^^^^^^",
+        "^^^^^^^^^^^^^^^^^^^^^^^^",
+        "^^^^^^^^^^^|||||||||||^^",
+        "^^^^^^^^^^^|__$$$$$_A|^^",
+        "^^^^^^^^^^^|&_______A|^^",
+        "^^^^^^^^^^^|&__eee__A|^^",
+        "^^^^^^^^^^^|&__eee__A|^^",
+        "^^^^^^^^^^^|________A|^^",
+        "^^^^^^^^^^^|T________|^^",
+        "^^^^^^^^^^^|||+||____|^^",
+        "^^^^^^^^^^^|___T|___R|^^",
+        "^^^^^^^^^^^|_h__|b__R|^^",
+        "^^^^^^^^^^^|rrrv||_RR|^^",
+        "^^|||||||||||||||<_|||^^",
+        "^^|UUU  ? = ZW|%||_|w|^^",
+        "^^|           |____+_|^^",
+        "^^|N          +__%%|w|^^",
+        "^^|N          ||||||||^^",
+        "^^|   ~~ zz   |^^^^^^^^^",
+        "^^|qqq~~gzzmmm|^^^^^^^^^",
+        "^^|||||||||||||^^^^^^^^^",
+        "^^^^^^^^^^^^^^^^^^^^^^^^",
+        "^^^^^^^^^^^^^^^^^^^^^^^^",
+        "^^^^^^^^^^^^^^^^^^^^^^^^"
+      ],
+      "terrain": {
+        "_": "t_floor",
+        "w": "t_floor",
+        "v": "t_floor",
+        "h": "t_floor",
+        "A": "t_floor",
+        "e": "t_floor",
+        "$": "t_floor",
+        "r": "t_floor",
+        "%": "t_floor",
+        "R": "t_floor",
+        "V": "t_floor",
+        "b": "t_floor",
+        "T": "t_floor",
+        "&": "t_floor"
+      },
+      "furniture": {
+        "!": "f_crate_c",
+        "$": "f_counter",
+        "%": "f_table",
+        "&": "f_glass_cabinet",
+        "=": "f_sink",
+        "~": "f_chair",
+        "?": "f_home_furnace"
+      },
+      "items": {
+        "$": { "item": "beer", "chance": 15 },
+        "&": { "item": "liquor_and_spirits", "chance": 50 },
+        "%": { "item": "table_foyer", "chance": 40 }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "om_terrain": "2storyModern02_1",
+    "weight": 250,
+    "method": "json",
+    "object": {
+      "palettes": [ "standard_domestic_palette", "standard_domestic_lino_bathroom", "standard_domestic_landscaping_palette" ],
+      "fill_ter": "t_floor_waxed",
+      "rows": [
+        "._______.p..............",
+        "._______..||oo||oo||..[.",
+        "._______.||Vysxxsyy||%..",
+        "._______.:         H:%..",
+        "._______.:y  T  ll H:%..",
+        "._______.:y  E     H:%..",
+        "._______.:y     HHHH:%..",
+        "._______.||RRRR RRRR|%..",
+        "._______..||||| |||||^..",
+        "..--------%%%%| rrry||..",
+        ".[-----------%o  h  V||.",
+        "..|::::|-----%o     VV|.",
+        ".|| h  ||----%| |||   o.",
+        ".|y ff y|::*::| |>||+||.",
+        ".| hffh      Ly + |S08o.",
+        ".|  ff          |||008|.",
+        ".o  h           <||t9||.",
+        ".o       A A  |o||||||,.",
+        ".|y    6 JJJ  o```````,.",
+        ".|| aa 6      ¶``G````-.",
```

Comment:
```suggestion
        ".|| aa 6       ``G````-.",
```
Need to replace this character with the correct one, or give it a definition

Reply:
That symbol is what I found in the palette for closed glass door. It spawns correctly, as per the ground floor screenshot.

Reply:
Ok, I'm home now and able to look into this a bit further but still not sure why that error is popping up, as that symbol is listed in standard_domestic_palette

![PALETTE - symbol](https://user-images.githubusercontent.com/56778776/189470937-081d6e59-f7f2-4cf5-88c0-8a90efb2179e.png)


And also is used in house_08, as called by the same palette

![house08](https://user-images.githubusercontent.com/56778776/189470980-09ab1366-3a83-4a61-b4cf-691c89285848.png)


It spawns perfectly in my game, so I'm baffled.

Reply:
Can confirm that  the symbol is in `standard_domestic_palette`